### DISCUSSION TITLE
最新版cesium与VcSelectionIndicator选择器不兼容了，出现如下图报错，大佬

### DISCUSSION THREAD

**Discussion by HHKMFZY** [zouyaoji/vue-cesium#607] [Q&A] [answered]
TypeError: Cesium.SceneTransforms.wgs84ToWindowCoordinates is not a function
![image](https://github.com/user-attachments/assets/523dc9c8-33a0-4b77-9c43-ab78e6a98cc2)

**Comment by Cocytus-kyon** [comment]
你试试先安装个低版本的cesium如`npm install cesium@1.120.0`，再把node_modules中的vue-cesium中的所有wgs84ToWindowCoordinates替换为worldToWindowCoordinates，用vscode单独打开这个包就可以搜索并全部替换。

**Reply by zouyaoji** [reply, marked as answer]
3.2.9版本会兼容处理
